x/bridge/keeper/genesis.go

```diff
@@ -0,0 +1,31 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v23/x/bridge/types"
+)
+
+// InitGenesis initializes the bridge module's state from a provided genesis state.
+func (k Keeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
+	k.CreateModuleAccount(ctx)
```

Comment:
```suggestion
	k.CreateModuleAccount(ctx)
```

I don't think this is right/needed, should just be defined here in app.go 

https://github.com/osmosis-labs/osmosis/blob/520973d8326a90d21dcd461212f90f549ee7b001/app/modules.go#L107-L135

Reply:
I guess this should go together with the wiring then. For now, I deleted CreateModuleAccount